website/src/theme/PdfButton/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+export function ensureJs(url) {
+	return new Promise((resolve, reject) => {
+		if (!document.getElementById(url)) {
+			const script = document.createElement('script')
+			script.type = 'text/javascript'
+			script.onload = resolve
+			script.onerror = reject
+			script.src = url
+			script.id = url
+			document.head.append(script)
+		} else {
+			resolve(true)
+		}
+	});
+}
+
+function download(id) {
+
+	ensureJs('https://cdnjs.cloudflare.com/ajax/libs/html2pdf.js/0.10.1/html2pdf.bundle.min.js').then(() => {
+		var element = document.getElementById(id)
+
+		html2pdf().set({
+			pagebreak: { mode: 'avoid-all', before: '#page2el' },
+			image: { type: 'png' },
```

Comment:
Can you try jpeg and quality: 1? surprisingly it seems to produce better quality than png